### ISSUE TITLE
Support macOS XCTest UI Tests

### ISSUE THREAD

**Issue by zorgiepoo** [travis-ci/travis-ci#7075]
I don't think I found this reported elsewhere (well, #4473 may be the closest related issue).

I can't run tests for a macOS project that has UI tests using XCTest (Xcode introduced this in macOS 10.11).

I run into this accessibility issue:

> Unable to run UI Tests because Xcode Helper does not have permission to use Accessibility.
To enable UI testing, go to the Security & Privacy pane in System Preferences, select the Privacy tab,
then select Accessibility, and add Xcode Helper to the list of applications allowed to use Accessibility.

Any way to add support for this? I'm not familiar with a command line way of giving Xcode Helper accessibility permission (that isn't a ugly hack blocked by Apple now). This is [an example](https://github.com/sparkle-project/Sparkle/pull/948) for a project I've tried adding automated UI tests for.

**Comment by wolffan** [comment]
I can't get Travis to run properly UI tests either (for XCTest).
They just fail with failed expectation

**Comment by ElDeveloper** [comment]
I believe this is still an issue.

**Comment by zorgiepoo** [comment]
If this still occurs for people, it would be best to open a new issue if they aren't going to re-open this one (I can't). And make sure it gets activity every 90 days or provide them feedback about their stale issue bot 😉.

**Comment by zorgiepoo** [comment]
@DrTorte

**Comment by DrTorte** [comment]
Reopened, thanks @zorgiepoo. I'll make a note to check internally. :)

**Comment by wolffan** [comment]
There seem to be a bug with the bot... @DrTorte  not sure if it's still an issue

**Comment by kornelski** [comment]
Still an issue
